File: dayThree/java/afernoonSession/ParkingLotTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class ParkingLotTest {
        
    @Test
    public void tesToAddCarWhenParkingLotIsEmptxy(){
        int number =  5;
        String expected = ParkingLot.park(number);
        assertEquals(expected, "car added successfully");
    }
    
    @Test
    public void tesToRemoveCarWhenParkingLotIsOccupied(){
        int number =  5;
        String expected = ParkingLot.remove(number);
        assertEquals(expected, "car removed successfully");
    }
    
    
}

File: dayThree/python/afternoonSession/parking_lot_test.py
from unittest import TestCase
from parking_lot import park
from parking_lot import remove

class ParkingLotTest(TestCase):
    
    def test_to_park_car_is_working(self):
        self.assertEqual(park(5), "car added successfully")
    
    def test_to_remove_car_is_working(self):
        self.assertEqual(remove(5), "car removed successfully")
